test/e2e/specs/site-editor/font-library.spec.js

```diff
@@ -86,4 +86,161 @@ test.describe( 'Font Library', () => {
 			).toBeVisible();
 		} );
 	} );
+
+	test.describe( 'When user has permissions to edit font families', () => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentythree' );
+		} );
+
+		test.beforeEach( async ( { admin, editor, page } ) => {
+			await admin.visitSiteEditor();
+			await editor.canvas.locator( 'body' ).click();
+			await page.getByRole( 'button', { name: /styles/i } ).click();
```

Comment:
The `page.getByRole` uses case-insensitive matching by default; there's no need for `RegExp`. I would recommend using actual label string where possible. It makes it easier to search/replace them in tests.

See: https://playwright.dev/docs/api/class-page#page-get-by-role-option-name

Reply:
Ah thanks, I didn't realise `getByRole` was already case-insensitive. I've changed this so the actual strings are used in this suite of tests: https://github.com/WordPress/gutenberg/pull/59316/commits/93350b24df67aa9396856107cf65205ccc5d62f4